src/MySqlConnector/Core/TextRow.cs

```diff
@@ -103,7 +103,7 @@ protected override object GetValueCore(ReadOnlySpan<byte> data, ColumnDefinition
 
 		case ColumnType.Decimal:
 		case ColumnType.NewDecimal:
-			return Utf8Parser.TryParse(data, out decimal decimalValue, out int bytesConsumed) && bytesConsumed == data.Length ? decimalValue : throw new FormatException();
+			return Encoding.UTF8.GetString(data);
```

Comment:
We can't change the default type of Decimal columns from `decimal` to `string`.

Reply:
Okay I have added it back

Reply:
@bgrainger Can you review the new iteration?